content/guides/box-ai/extract-metadata-structured.md

```diff
@@ -0,0 +1,66 @@
+---
+rank: 9
+related_endpoints:
+  - post-ai-extract
+related_guides:
+  - box-ai/prerequisites
+  - box-ai/extract-metadata
+  - box-ai/get-agent-default-config
+---
+
+# Extract metadata from file (structured)
+
+<Message type="notice">
+Box AI Platform API is currently a beta feature offered subject to Box’s Main Beta Agreement, and the available capabilities may change. Box AI Platform API is available to all Enterprise Plus customers.
+
+</Message>
+
+With Box AI Platform API, you can extract metadata from the provided file
+and return it in the form of key-value pairs.
+Use this endpoint if you already have a metadata template that
+you can use to obtain data.
```

Comment:
I think I'd maybe add, an or statement that you don't have to have the metadata template? Or mention the metadata template as a possible use case? I think Scott mentioned that very few customers actually use the metadata template for this endpoint.

Reply:
Sure, will do :)